Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Build the manager binary
-FROM golang:1.15 as builder
+FROM golang:1.16 as builder
```

Comment:
From now onwards let's use bundle as a type for commit msg for all DOCKEFILE related changes.

Reply:
I can do that now

Reply:
Done.